jsonnet/telemeter/rules.libsonnet

```diff
@@ -57,7 +57,7 @@
             {
               record: 'id_install_type',
               expr: |||
-                0 * (count by (_id, install_type) (label_replace(label_replace(label_replace(label_replace(topk by (_id) (1, cluster_installer), "install_type", "upi", "type", "other"), "install_type", "ipi", "type", "openshift-install"), "install_type", "hive", "invoker", "hive"), "install_type", "assisted-installer", "invoker", "assisted-installer")) or on(_id) (label_replace(count by (_id) (cluster:virt_platform_nodes:sum), "install_type", "hypershift-unknown", "install_type", ""))*0)
+                0 * (count by (_id, install_type) (label_replace(label_replace(label_replace(label_replace(label_replace(topk by (_id) (1, cluster_installer), "install_type", "upi", "type", "other"), "install_type", "ipi", "type", "openshift-install"), "install_type", "hive", "invoker", "hive"), "install_type", "assisted-installer", "invoker", "assisted-installer"), "install_type", "infrastructure-operator", "invoker", "assisted-installer-operator") or on(_id) (label_replace(count by (_id) (cluster:virt_platform_nodes:sum), "install_type", "hypershift-unknown", "install_type", ""))*0)
```

Comment:
Why did you pick “infrastructure-operator”?  Why not “assisted-install-something”?  The former is very generic.

Reply:
@smarterclayton It was a corporate marketing decision

Reply:
we the developers are just aligning with that :)

Reply:
The first name was `assisted-installer-operator` but it was rejected by the PM.
Please see https://coreos.slack.com/archives/C01FJCM6MBQ/p1624274468046800?thread_ts=1624257563.042500&cid=C01FJCM6MBQ

Reply:
This is an invalid promQL expression as you opened parenthesis with `label_replace(` but never closed it.

Let's rewrite it into multiple lines for readability:
```
(
  count by (_id, install_type) (
    label_replace(
      label_replace(
        label_replace(
          label_replace(
            label_replace(
              topk by (_id) (1, cluster_installer), "install_type", "upi", "type", "other"            
            ), "install_type", "ipi", "type", "openshift-install"
          ), "install_type", "hive", "invoker", "hive"
        ), "install_type", "assisted-installer", "invoker", "assisted-installer"
      ), "install_type", "infrastructure-operator", "invoker", "assisted-installer-operator" 
    )
  ) or on(_id) (
    label_replace(
      count by (_id) (cluster:virt_platform_nodes:sum), "install_type", "hypershift-unknown", "install_type", ""
    )
  ) * 0 
) * 0
```